test/extended/baremetal/hosts.go

```diff
@@ -186,4 +186,78 @@ var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] Baremetal platfor
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(found).To(o.BeFalse())
 	})
+
+	g.It("configure BIOS settings during cleaning", func() {
+		var procTurboMode = "ProcTurboMode"
+
+		skipIfNotBaremetal(oc)
+
+		dc := oc.AdminDynamicClient()
+		hostData, secretData := helper.GetExtraWorkerData(0)
+
+		// Deploy extra worker and wait
+		host, _ := helper.CreateExtraWorker(hostData, secretData)
+		host = helper.WaitForProvisioningState(host, "available")
```

Comment:
I think, `helper.DeployExtraWorker` does the same thing above 3 lines do.

Reply:
Thanks Arda, good point. I will change to use DeployExtraWorker.